example/styleguide.json

```diff
@@ -1,5 +1,9 @@
 {
   "title": "React Style Guide",
+  "typekit": {
+    "id": "rtq7sdx",
```

Comment:
That was not my point. I meant that the example can reflect many features of the tool, and this one is not one that is required to be in the example